src/azure-cli/az.ps1

```diff
@@ -0,0 +1,225 @@
+$env:AZ_INSTALLER="PIP"
+
+if (Test-Path "$PSScriptRoot\python.exe") {
+    # Perfer python.exe in venv
+    & "$PSScriptRoot\python.exe" -m azure.cli $args
+}
+else {
+    # Run system python.exe
+    python.exe -m azure.cli $args
+}
+
+# SIG # Begin signature block
+# MIInqgYJKoZIhvcNAQcCoIInmzCCJ5cCAQExDzANBglghkgBZQMEAgEFADB5Bgor
+# BgEEAYI3AgEEoGswaTA0BgorBgEEAYI3AgEeMCYCAwEAAAQQH8w7YFlLCE63JNLG
```

Comment:
There is one problem with `az.ps1` for `pip`. 

- ESRP code-sign was executed on Azure DevOps's Windows agent. When `git` checks out the source code on Windows, the end of line sequence is `CRLF` (`\r\n`). This is the default of Windows `git`:
   ![image](https://user-images.githubusercontent.com/4003950/185730246-608b7fcd-fa45-42fa-88bb-eed867b487af.png)
   so the signature was generated for the `CRLF` version of `az.ps1`. 
- However, wheels are built on Linux agent, converting `az.ps1`'s end of line sequence to `LF` (`\n`). 
- Even when installing with `pip -e`, `pip` converts `CRLF` to `LF` (for simplicity I guess).

Thus, the signature doesn't match the file content anymore.